### ISSUE TITLE
cgi.cfg doesn't update with different user

### ISSUE THREAD

**Issue by rkaramandi** [JasonRivers/Docker-Nagios#114]
When you modify the username to be a different user (in my case I changed the environment variable `NAGIOSADMIN_USER`), the cgi.cfg still assumes the default username (`nagiosadmin`). This makes it so that you can log in, but you can't see any of the pages correctly; you get an error on the page like:

```
It appears as though you do not have permission to view information for any of the services you requested...

If you believe this is an error, check the HTTP server authentication requirements for accessing this CGI
and check the authorization options in your CGI configuration file.
```

If you can run a command to update the `nagiosadmin` user to `${NAGIOSADMIN_USER}` by using `sed`, that would make it more flexible, and avoid having this issue. Basically add this at the bottom of the file to substitute the environment variable instead of keeping the static `nagiosadmin` user.

Basically, adding this in the Dockerfile would help:

```dockerfile
RUN sed -i "s/nagiosadmin/$NAGIOSADMIN_USER/g" /opt/nagios/etc/cgi.cfg
```
